ui/src/routes/organizations/$orgId/products/$productId/repositories/$repoId/_repo-layout/create-run/-create-run-utils.ts

```diff
@@ -33,6 +33,13 @@ const packageManagerOptionsSchema = z.object({
   options: z.array(keyValueSchema).optional(),
 });
 
+const reporterOptionsSchema = z.object({
+  options: z.object({
+    outputFileFormats: z.string().optional(),
+    templateIds: z.string().optional(),
```

Comment:
Just a remark: IMO it's a bit weird to combine properties that are supported by different reporters into a single options schema. Ideally, each reporter should have its own schema, as it's more or less just "coincidence" that both CycloneDX and SPDX reporters support the equally named "outputFileFormats" option. But I'm not sure whether it's worth the effort.